code/modules/power/tesla/energy_ball.dm

```diff
@@ -40,15 +40,16 @@ var/list/blacklisted_tesla_types = list(/obj/machinery/atmospherics,
 	if(!is_orbiting)
 		handle_energy()
 		var/amount_to_move = 2 + orbiting_balls.len * 2
+		var/what_does_the_scouter_say_about_the_balls_power_level = (TESLA_DEFAULT_POWER + (TESLA_MINI_POWER * orbiting_balls.len))
```

Comment:
It really doesn't, to be honest; it's just really high.

It does matter a bit more, on our codebase, where we have arc flashing and stuff, above a certain level--but even then, it's pretty trivial.